backend/src/main/java/com/sim_backend/websockets/messages/BootNotificationResponse.java

```diff
@@ -0,0 +1,40 @@
+package com.sim_backend.websockets.messages;
+
+import com.google.gson.annotations.SerializedName;
+import com.sim_backend.websockets.OCPPMessage;
+import com.sim_backend.websockets.OCPPMessageInfo;
+
+@OCPPMessageInfo(
+    messageCallID = OCPPMessage.CALL_ID_RESPONSE,
+    messageName = "BootNotificationResponse")
+public class BootNotificationResponse extends OCPPMessage {
+
+  @SerializedName("status")
+  private String status; // Status of the BootNotification (Accepted, Rejected)
+
+  @SerializedName("interval")
+  private int interval; // Interval (time in seconds) for next BootNotification
+
+  // Constructor, getters, setters
+  public BootNotificationResponse(String status, int interval) {
```

Comment:
Add basic validation for message fields (especially response messages). While we can catch some issues with schema validation, something like a negative interval can slip through currently.